### DISCUSSION TITLE
Does ultima-ng-16.0.0 using primeng: 16.9.8-lts prohibit me from viewing the app in my phone?

### DISCUSSION THREAD

**Discussion by cydriclopez** [primefaces/community#1741] [PrimeNG Templates] [answered]
I have recently subscribed to primeng lts v16. Does ultima-ng-16.0.0 template, using primeng: 16.9.8-lts, prohibit me from viewing the app in my phone?

I get the red banner when I view the ultima-ng-16.0.0 template from my phone. I view the app using an incognito tab, for better refresh & hard reload, in my android phone hitting my laptop's ip address via an open port. I have deleted the dist/ultima-ng folder and recreated via "ng build". Am I missing something? Thanks.

![Screenshot from 2024-04-22 19-16-18](https://github.com/primefaces/community/assets/8703893/944c0133-8911-495d-b5b5-5c5c9ceab5a7)

**Comment by cetincakiroglu** [comment]
You need to verify the LTS. Please see the `Usage` section in https://primeng.org/lts

**Reply by cydriclopez** [reply, marked as answer]
Thanks for your reply. I created a self-signed openssl certificate and serve https in my laptop. The red banner went away in my phone. Your LicenseManager.verify(LICENSE_KEY, PASS_KEY) requires https connection. Problem fixed. It was my fault.
